internal/model/user.go

```diff
@@ -0,0 +1,10 @@
+package model
+
+
+
+type User struct {
+	Id			int64       
+	SlackId		string       
```

Comment:
Nothing critical, but this field could be called `SlackID`

```suggestion
	SlackID		string       
```

Reply:
@tiagotor  change in this files as well: 

- internal/commands/user.go

- internal/commands/user_test.go